src/lwip/BUILD.gn

```diff
@@ -202,8 +203,7 @@ if (current_os == "zephyr" || current_os == "mbed") {
       public_deps += [ "${ti_simplelink_sdk_build_root}:ti_simplelink_sdk" ]
     } else if (lwip_platform == "cc32xx") {
       public_deps += [ "${ti_simplelink_sdk_build_root}:ti_simplelink_sdk" ]
-      sources +=
-          [ "${chip_root}/third_party/lwip/repo/lwip/src/apps/mdns/mdns.c" ]
+      sources += [ "${chip_root}/third_party/lwip/repo/src/apps/mdns/mdns.c" ]
```

Comment:
lwip mDNS is known to be incompatible with Matter. Not sure why this is in.

Reply:
removed